package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "prepare": "husky install",
     "build": "nx run-many --target=build",
-    "lint": "nx run-many --target=lint",
+    "lint": "nx run-many --target=lint --quiet",
```

Comment:
Added `--quiet`, which will omit warnings and only show lint errors. This will make reading the CI output a bit more useful.